### ISSUE TITLE
3.2 RC1 Transparent material with opaque prepass, bug in alpha channel

### ISSUE THREAD

**Issue by fbcosentino** [godotengine/godot#35401]
Godot version: Godot 3.2 RC1

OS/device including version: Windows 10

Issue description: Setting a material as transparent and opaque prepass, alpha channel behaves weird. This was working perfectly on 3.1 and 3.2 the behaviour is shown in the attached gif:
![opaque_prepass_transparent](https://user-images.githubusercontent.com/28926813/72806143-749f4f00-3c4c-11ea-9943-c660bb36572e.gif)

Steps to reproduce: Create a cube, create a SpatialMaterial. Set flag Transparent to True, set Opaque pre-pass. In albedo, play with alpha channel slider

Minimal reproduction project: N/A

**Comment by Calinou** [comment]
Can you reproduce this in any of the earlier Godot 3.2 betas and alphas?

**Comment by clayjohn** [comment]
I can reproduce but only on the GLES2 backend.

**Comment by Tooniis** [comment]
I'm experiencing this bug too, but it doesn't happen in the material preview
![Screenshot from 2020-09-06 15-28-46](https://user-images.githubusercontent.com/36982494/92324914-30d04d00-f057-11ea-85b7-2840127f15be.png)
![Screenshot from 2020-09-06 15-28-43](https://user-images.githubusercontent.com/36982494/92324915-32017a00-f057-11ea-91a5-56ccd4961830.png)
Does the material preview always use the GLES3 renderer or something?

**Comment by Calinou** [comment]
> Does the material preview always use the GLES3 renderer or something?

No, as the whole engine can only use one renderer at a given time.
